src/PointwiseFunctions/Hydro/EquationsOfState/Python/Bindings.cpp

```diff
@@ -7,7 +7,14 @@
 #include "PointwiseFunctions/Hydro/EquationsOfState/Python/PiecewisePolytropicFluid.hpp"
 #include "PointwiseFunctions/Hydro/EquationsOfState/Python/PolytropicFluid.hpp"
```

Comment:
[Commit message]
This isn't a `PiecewisePolynomial` it's a `PiecewisePolytrope`

Reply:
🤦‍♂️ fixed